ejb/src/main/java/org/wildfly/httpclient/ejb/Serializer.java

```diff
@@ -81,18 +81,18 @@ static void serializePackedInteger(final ObjectOutput out, int value) throws IOE
         }
     }
 
-    static int deserializePackedInteger(final ObjectInput input) throws IOException {
-        int ret = input.readByte();
+    static int deserializePackedInteger(final ObjectInput in) throws IOException {
```

Comment:
Is "in" an integer, an object input or an inverter? I get it that we can use in for input and out for output, but a lot of context is lost.

Reply:
Fixed.